### ISSUE TITLE
Reading: Contextual Embeddings: When Are They Worth It?

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#146]
## 0. Paper
@inproceedings{arora-etal-2020-contextual,
    title = "Contextual Embeddings: When Are They Worth It?",
    author = "Arora, Simran  and
      May, Avner  and
      Zhang, Jian  and
      R{\'e}, Christopher",
    booktitle = "Proceedings of the 58th Annual Meeting of the Association for Computational Linguistics",
    month = jul,
    year = "2020",
    address = "Online",
    publisher = "Association for Computational Linguistics",
    url = "https://www.aclweb.org/anthology/2020.acl-main.236",
    doi = "10.18653/v1/2020.acl-main.236",
    pages = "2650--2663",
    abstract = "We study the settings for which deep contextual embeddings (e.g., BERT) give large improvements in performance relative to classic pretrained embeddings (e.g., GloVe), and an even simpler baseline{---}random word embeddings{---}focusing on the impact of the training set size and the linguistic properties of the task. Surprisingly, we find that both of these simpler baselines can match contextual embeddings on industry-scale data, and often perform within 5 to 10{\%} accuracy (absolute) on benchmark tasks. Furthermore, we identify properties of data for which contextual embeddings give particularly large gains: language containing complex structure, ambiguous word usage, and words unseen in training.",
}
## 1. What is it?
They compare the random, non-contextual word embeddings and contextual language models in **the amount of data** and **language property**.

### 4.1 Impact of Training Data Volume
![スクリーンショット 2021-02-27 15 46 07](https://user-images.githubusercontent.com/45454055/109378068-e9928d00-7912-11eb-89c6-4dfd4b9d6532.png)

### 4.2 
![スクリーンショット 2021-02-27 15 52 15](https://user-images.githubusercontent.com/45454055/109378179-c74d3f00-7913-11eb-8f99-71becb208b3b.png)
